content/basics/basics.mdx

```diff
@@ -0,0 +1,327 @@
+---
+name: Basics
+route: /basics
+---
+
+import { Playground } from 'docz'
+
+# Basic Introduction
+
+[MobX](https://mobx.js.org/README.html) lets you create observable state. This
+library, `mobx-react` integrates it with React. It provides a way for you to
+create React components that respond to MobX state changes.
+
+> Frameworks such as [mobx-state-tree](https://mobx-state-tree.js.org) and
+> [mobx-keystone](https://mobx-keystone.js.org/) build on
+> top of MobX to offer a higher-level way to create observable models. These models
+> give you additional properties, such as automatic serialization to JSON. This
+> introduction uses plain MobX models. The principles of React
+> integration are the same for these advanced libraries are the same
```

Comment:
This ending of this sentence doesn't make much sense, try to rewrite it differently.